config.json

```diff
@@ -1946,6 +1948,12 @@
       "name": "Exceptions",
       "blurb": "All errors in Elixir implement the Exception Behaviour. It provides an easy way to create your own custom errors."
     },
+    {
+      "uuid": "da243817-5c15-461c-8305-dba188b6a527",
+      "slug": "file",
+      "name": "File",
+      "blurb": "Functions for working with files are provided by the File module."
+    },
```

Comment:
No change here, just moved it down in the list so that it's sorted alphabetically.

Reply:
:+1: